amundsen_application/static/js/components/TableDetail/TableIssues/styles.scss

```diff
@@ -34,10 +31,10 @@
     justify-content: flex-end;
     flex-grow: 1.1;
     min-width: fit-content;
-    margin: 0px 0px 0px 5px;
+    margin-bottom: $spacer-1; 
```

Comment:
Did you mean for this to be `margin-left`? The status is squished up against the text.
![image](https://user-images.githubusercontent.com/1790900/77595462-75f03080-6eb6-11ea-9943-1476779c3a7f.png)

Also in this case I think `margin-left: $spacer-2` would make things even easier to read as well, but up to you based on what looks closer to the designs.

Reply:
good catch - i wasnt testing overflow in this round